recipes/nose-exclude/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "nose-exclude" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f78fa8b41eeb815f0486414f710f1eea0949e346cfb11d59ba6295ed69e84304
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - nose
+    - pip
+    - python
+    - setuptools
+  run:
+    - nose
+    - python
+    - setuptools
+
+test:
+  imports:
+    - nose_exclude
+
+about:
+  home: https://github.com/kgrandis/nose-exclude
+  license: GNU Library or Lesser General Public (LGPL)
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: LGPL-2.1
```